Sunrise/Cells/CartItemCell.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  CartItemCell.swift
+//  Sunrise
+//
+//  Created by Nikola Mladenovic on 5/31/16.
+//  Copyright © 2016 Commercetools. All rights reserved.
+//
+
+import UIKit
+
+class CartItemCell: UITableViewCell {
```

Comment:
Can we name it `CartLineItemCell` or similar? I just want to use the word `LineItem`, which is how we refer to those items. That's of course if I get right this class. By its attributes it looks like line item, although I see quantity is not there.